components/bms_worker/include/Module.h

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file Module.h
+ * @brief  Header file for Module Manager
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @date 2023-12-28
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "ModuleDesc.h"
+
+#include "stdint.h"
+
+
+/******************************************************************************
+ *                              E X T E R N S
+ ******************************************************************************/
+
+/**< Modules */
+extern const ModuleDesc_S BMS_desc;
+extern const ModuleDesc_S Cooling_desc;
```

Comment:
module names should be either acronyms or abbreviations. eg. COOL and ENV instead of Cooling and Environment. This will make it easier to name variables and functions in such a way that the namespace is clear